Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDateExtractorConfiguration.cs

```diff
@@ -1,151 +1,149 @@
-﻿using System.Text.RegularExpressions;
-using System.Collections.Generic;
-using Microsoft.Recognizers.Text.Number.Spanish.Extractors;
-using Microsoft.Recognizers.Text.Number.Parsers;
-using Microsoft.Recognizers.Text.Number.Spanish.Parsers;
-using Microsoft.Recognizers.Text.DateTime.Extractors;
-
-namespace Microsoft.Recognizers.Text.DateTime.Spanish.Extractors
```

Comment:
It is kind of annoying there are all these large diffs.  Was it only space differences or something?